src/signs/ui/search/MesZLContent.tsx

```diff
@@ -55,75 +26,53 @@ export default function MesZlContent({
   mesZl: string
   cutOff?: number
 }): JSX.Element | null {
-  const [mesZlHead, setMesZlHead] = useState<string>('')
-  const [mesZlBody, setMesZlBody] = useState<string>('')
-
-  const [ready, setReady] = useState(false)
-
   const { mesZlHeadMarkdown, mesZlBodyMarkdown } = splitMesZl(mesZl, cutOff)
+  return (
+    <>
+      <div>
+        <pre>
+          <ContainerWithInnerHtml
+            className={'text-center'}
+            markdown={mesZlHeadMarkdown}
+          />
+        </pre>
+      </div>
+      <ContainerWithInnerHtml markdown={mesZlBodyMarkdown} />
 
-  useEffect(() => {
-    ;(async () => {
-      setMesZlHead(await convertMarkdownToHtml(mesZlHeadMarkdown))
-      setMesZlBody(await convertMarkdownToHtml(mesZlBodyMarkdown))
-
-      setReady(true)
-    })()
-  }, [mesZl, mesZlHeadMarkdown, mesZlBodyMarkdown])
-
-  if (ready) {
-    return (
-      <>
-        <div>
-          <pre>
-            <div
-              className="text-center"
-              dangerouslySetInnerHTML={{
-                __html: DOMPurify.sanitize(mesZlHead),
-              }}
-            />
-          </pre>
-        </div>
-        <div
-          dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(mesZlBody) }}
-        />
-        {mesZl.split('\n').length > cutOff && (
-          <div className="text-center">
-            <br />
-            <strong>
-              <Link to={`/signs/${encodeURIComponent(signName)}`}>
-                (Read more ...)
-              </Link>
-            </strong>
-
-            <br />
-            <br />
-          </div>
-        )}
-        <div className="text-center border border-dark">
-          <strong>From</strong>
-          <br />
-          R. Borger,{' '}
-          <em>
-            Mesopotamisches Zeichenlexikon. Zweite, revidierte und aktualisierte
-            Auflage.&nbsp;
-          </em>
-          Alter Orient und Altes Testament 305.
-          <br /> Münster: Ugarit-Verlag, <sup>2</sup>2010; Kapitel &#8546;.
+      {mesZl.split('\n').length > cutOff && (
+        <div className="text-center">
           <br />
+          <strong>
+            <Link to={`/signs/${encodeURIComponent(signName)}`}>
+              (Read more ...)
+            </Link>
+          </strong>
+
           <br />
-          <strong>By permission from Ugarit-Verlag.</strong>
           <br />
-          <ExternalLink
-            className="text-dark "
-            href="https://ugarit-verlag.com/en/products/0e8e7ca5d1f5493aa351e3ebc42fb514"
-          >
-            <i className="pointer__hover my-2 fas fa-shopping-cart fa-2x" />
-          </ExternalLink>
         </div>
-      </>
-    )
-  } else {
-    return null
-  }
+      )}
+      <div className="text-center border border-dark">
+        <strong>From</strong>
```

Comment:
Should this be a heading and not `strong`?

Reply:
strong seems correct i checked with the pdf from enrique.

Reply:
Why is strong correct? The PDF has nothing to do with semantic HTML. Strong, em etc. should not be used for styling. There is CSS for that.

Reply:
ok